skyramp/management-plane/targets/grafana.yaml

```diff
@@ -0,0 +1,7 @@
+namespace: skyramp
+containers:
+  - type: helm
+    releaseName: skyramp-grafana-dashboard
+    chartRepository: https://letsramp.github.io/helm/
```

Comment:
The Prometheus operator helm chart is using the helm object {{ Release.Namespace }} to reference services managed by MutatingWebhookConfiguration and ValidatingWebhookConfiguration. We need to add 
releaseNamespace in the target description for grafana.yaml.

Note: this is updated in the PR for worker.

Reply:
thanks for flagging; done